Sources/AsyncHTTPClient/HTTPClientProxyHandler.swift

```diff
@@ -24,11 +24,19 @@ import NIOHTTP1
 /// If a `TLSConfiguration` is used in conjunction with `HTTPClientProxy`,
 /// TLS will be established _after_ successful proxy, between your client
 /// and the destination server.
-public extension HTTPClient {
+public extension HTTPClient.Configuration {
```

Comment:
The comments above refer to `HTTPClientProxy` which is out of date.  Probably the comments should be on `struct Proxy` not the extension too?

Reply:
@ianpartridge see https://github.com/swift-server/async-http-client/pull/90/commits/e0eb49647e606d86d414540aeac4a21c69816e7f wdyt?

Reply:
Great, thanks.